core_java/grep/grep.iml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This, as well as the .idea folder is IntelliJ specific and should not be a part of the git repo (it should be ignored).